plugins/docker/stats_pumper.go

```diff
@@ -0,0 +1,393 @@
+package docker
```

Comment:
I feel like the filename should probably be something like `stats_handling.go` or something along those lines since it sounds like a type name, but then the file doesn't have a type by that name.